CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Thank you for taking the time to contribute to JustiFi Web Components! The following is a set of guidelines for contributing to this project.
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Important Links
+ - [Web Component Documentation](https://storybook.justifi.ai/?path=/docs/introduction--docs)
+ - [JustiFi API Documentation](https://developer.justifi.ai/)
+
+## How to Report a Bug
+
+1. Create an Issue for the Web Component Repo using the Repo's Issue Page
+2. In the body of the Issue, enter `/` to select our Bug Reporting Template and fill out each section as required. 
+
+Note: Please include as many details as possible to assist our team in quickly identifying and resolving the issue. 
+
+## How to Request an Enhancement
+
+1. Create an Issue for the Web Component Repo using the Repo's Issue Page
+2. In the body of the Issue, enter `/` to select our Enhancement Request Template and fill out each section as required. 
+
+Note: Please include as many details as possible to help our team understand the intent behind your request, and how this request will potentially improve the user experience. 
```

Comment:
I wonder if we should expand this to `and how this request will potentially improve the user experience for all consumers of this project` or something to that affect